cpp/examples/tutorial_examples/arrow_example.cc

```diff
@@ -0,0 +1,94 @@
+#include <arrow/api.h>
+#include <arrow/result.h>
+#include <arrow/status.h>
+#include <arrow/table.h>
+
+#include <iostream>
+
+using arrow::Status;
+
+namespace { 
```

Comment:
Probably we should not use anonymous namespaces or namespaces in general for these representative examples.

Reply:
Went ahead and fixed that up.